tsconfig.app.json

```
@@ -2,13 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "./out-tsc/app",
-    "types": []
+    "types": ["node"]
```

```suggestion
    "types": []
```
For me it's working without this in the current state. If it still gives you an issue try removing node_modules and re-running `npm i` (with the updated `package.json` that does not include that extra package)